salt/grains/core.py

```diff
@@ -1156,6 +1156,23 @@ def os_data():
         grains.update(_windows_cpudata())
         grains.update(_windows_virtual(grains))
         grains.update(_ps(grains))
+
+        if 'Server' in grains['osrelease']:
```

Comment:
Windows servers have `Server` in their osrelease and nonserver windows systems do not.